cfme/v2v/infrastructure_mapping.py

```diff
@@ -0,0 +1,397 @@
+    source_datastores = MultiSelectList("source_datastores")
+    target_datastores = MultiSelectList("target_datastores")
+    cluster_selector = BootstrapSelect(id="cluster_select")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.source_datastores.is_displayed
+            and self.target_datastores.is_displayed
+            and (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0)
+        )
+
+    def fill(self, values):
+        """Use to add all mappings specified in values.
+        Args:
+            values:
+                format as following:
+                    {
+                        'cluster_name': {
+                            'mappings': [
+                                {
+                                    'sources':['item1','item2'],
+                                    'target':['item_target']
+                                },
+                                ...
+                            ]
+                        },
+                        ...
+                    }
+        """
+        for cluster in values if isinstance(values, dict) else []:
+            if self.cluster_selector.is_displayed:
+                self.cluster_selector.fill(cluster)
+            for mapping in values[cluster]["mappings"]:
+                self.source_datastores.fill(mapping["sources"])
+                self.target_datastores.fill(mapping["target"])
+                self.add_mapping.click()
+        was_change = not self.mappings_tree.is_empty
+        self.after_fill(was_change)
+        return was_change
+
+
+class InfraMappingWizardNetworksView(InfraMappingForm):
+    include_buttons_set2 = View.include(InfraMappingWizardCommon)
+    source_networks = MultiSelectList("source_networks")
+    target_networks = MultiSelectList("target_networks")
+    next_btn = Button("Create")  # overriding, since 'Next' is called 'Create' in this form
+    cluster_selector = BootstrapSelect(id="cluster_select")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.source_networks.is_displayed
+            and self.target_networks.is_displayed
+            and (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0)
+        )
+
+    def fill(self, values):
+        """Use to add all mappings specified in values.
+        Args:
+            values:
+                format as following:
+                    {
+                        'cluster_name': {
+                            'mappings': [
+                                {
+                                    'sources':['item1','item2'],
+                                    'target':['item_target']
+                                },
+                                ...
+                            ]
+                        },
+                        ...
+                    }
+        """
+        for cluster in values if isinstance(values, dict) else []:
```

Comment:
Q: values Arg specifies `dict` then why need `list`. I think better to restrict user for `dict`.